README_DEV.md

```diff
@@ -0,0 +1,258 @@
+  - Run `python harvester/main.py pumps.geojson` to generate a `pumps.geojson` file
+  - Upload this file manually to the Supabase bucket (http://localhost:54323/project/default/storage/buckets/data_assets)
+  - Copy the Supabase URL of the uploaded file for later use as `VITE_MAP_PUMPS_SOURCE_URL` variable
+- Download the Geojson file for Berlin districts:
+  - Go to https://github.com/funkeinteraktiv/Berlin-Geodaten/blob/master/berlin_bezirke.geojson and download the GeoJSON manually
+  - Upload the file to your Supabase instance at http://localhost:54323/project/default/storage/buckets/data_assets
+  - Copy the URL of the uploaded file for later use as `VITE_BEZIRKE_URL` variable
+- Change directory to `giessdenkiez-de-postgres-api/supabase` to start the Supabase Edge functions
+  - `cp .env.sample .env` and change the variables:
+    - `PUMPS_URL=<url_to_your_pumps_file>` set value to the URL of your pumps file
+  - Run `supabase functions serve --env-file supabase/.env` in `giessdenkiez-de-postgres-api` directory
+- Setup the Frontend:
+  - Change directory to `giessdenkiez-de` repository
+  - `nvm install && nvm use`
+  - `npm ci`
+  - `cp .env.sample .env`
+  - `direnv allow`
+  - `npm run dev`
+  - Open `http://localhost:5173/` in the browser, you will see an eternal loading screen indicating that trees are loading. In the browser console, you will see many errors. **This is expected**, continue with the next setup steps.
+  - Prepare the correct values in the `.env` file:
+    - `VITE_MAPBOX_API_KEY=<your-mapbox-api-key>` set to your Mapbox Frontend access token
+    - `VITE_MAPBOX_TREES_TILESET_URL=<your-trees-tileset-url>` set to the Mapbox tileset URL, e.g. `VITE_MAPBOX_TREES_TILESET_URL=mapbox://<account_name>.<tileset_id>` use your Mapbox account name and tileset ID.
+    - `VITE_MAP_PUMPS_SOURCE_URL=<your-pumps-source-url>` the URL to your `pumps.geojson` file, as previously uploaded
+    - `VITE_BEZIRKE_URL=<your_bezirke_geojson_url>` set to the URL of the previously generated and uploaded GeoJson file of the districts
+  - `direnv allow` to reload the environment
+  - `npm run dev` to reload the App
+  - Open `http://localhost:5173/map?treeId=00008100%3A001f2573` in the browser, you should see a map zoomed to a specific tree. 🎉
+
+## Step 2: Adapt to your city
+
+### After executing the following steps, you will have a working version of Gieß den Kiez (for your own city) running locally.
+
+- In the `giessdenkiez-de-postgres-api` directory, do the following:
+  - Connect to your local Gieß den Kiez database and delete all rows in all tables in the `public` scheme
+    - either manually
+    - or by removing the `seed.sql` file in the `giessdenkiez-de-postgres-api/supabase` directory and restarting Supabase via `npx supabase stop` and `npx supabase start` and `npx supabase db reset`
```

Comment:
suggestion: use `npx supabase stop --no-backup` to make sure nothing goes wrong (it did for me a few times)